src/Query/FilterParser.ts

```diff
@@ -34,6 +34,7 @@ import { BacklinkField } from './Filter/BacklinkField';
 // When adding new fields keep this order in mind, putting fields that are more specific before fields that
 // may contain them, and keep BooleanField last.
 export const fieldCreators: EndsWith<BooleanField> = [
+    // NEW_QUERY_INSTRUCTION_EDIT_REQUIRED
```

Comment:
Why would we need to add a new query instruction in the list of the field creators?

Reply:
Shall this be `NEW_TASK_FIELD_EDIT_REQUIRED`?

Reply:
> Shall this be `NEW_TASK_FIELD_EDIT_REQUIRED`?

I think this was the one I went backwards and forwards on a bit... My conclusion was that `NEW_QUERY_INSTRUCTION_EDIT_REQUIRED` was more relevant....

If you look at the changes to the Contributing pages linked in the description of this PR, the steps are likely to be:

1. Add the new field - by searching for `NEW_TASK_FIELD_EDIT_REQUIRED` - which is a huge chunk of work already
2. Then, add new query instructions, as later commits or even later PR - using `NEW_QUERY_INSTRUCTION_EDIT_REQUIRED`...

Reply:
Oh yes indeed.

Another option is to add both comments.

Reply:
I couldn't think of anything that needed to be done here when adding a new value to Task, though, so what's the point...

What have you spotted that I haven't

Reply:
No, sorry, you are right, no need to change.